MinecraftClient/ChatBot.cs

```diff
@@ -762,6 +762,16 @@ protected void LogDebugToConsole(object text)
                 LogToConsole(text);
         }
 
+        protected void LogToConsoleTranslated(string key, params object[] args)
```

Comment:
Methods in `ChatBot.cs` need a `/// Documentation` even if the name is rather obvious here 😉